app/__init__.py

```diff
@@ -19,7 +19,34 @@
     allow_origin_regex=settings.origins_regex
 )
 
-app.include_router(router)
+app.include_router(
+    users.fastapi_users.get_register_router(),
+    prefix="/auth",
+    tags=["auth"],
+)
+app.include_router(
+    users.fastapi_users.get_users_router(),
+    prefix="/auth",
+    tags=["users"],
+)
+app.include_router(
+    users.fastapi_users.get_auth_router(jwt_authentication),
+    prefix="/auth",
+    tags=["auth"],
+)
+app.include_router(
+    users.fastapi_users.get_reset_password_router(),
+    prefix="/auth",
+    tags=["auth"],
+)
+app.include_router(
+    users.fastapi_users.get_oauth_router(google_oauth_client, settings.secret),
+    prefix="/auth/google",
```

Comment:
Is there a signout if I login with Google?

Reply:
There's a generic loggout endpoint, it can also be done from the client side